arviz/plots/plot_utils.py

```diff
@@ -627,3 +630,34 @@ def filter_plotters_list(plotters, plot_kind):
         )
         return plotters[:max_plots]
     return plotters
+
+
+def get_plotting_method(plot_name, plot_module, backend):
+    """Returns plotting function for correct backend"""
+    _backend = {"mpl": "matplotlib", "bokeh": "bokeh", "matplotlib": "matplotlib"}
+
+    try:
+        backend = _backend[backend]
+    except KeyError:
+        raise KeyError(
+            "Backend {} is not implemented. Try backend in {}".format(backend, set(_backend.values))
+        )
+
+    if backend == "bokeh":
+        try:
+            import bokeh
+
+            assert bokeh.__version__ >= "1.4.0"
+        except (ImportError, AssertionError):
+            raise ImportError("'bokeh' backend needs Bokeh (1.4.0+) installed.")
+
+    module = importlib.import_module(
+        "arviz.plots.backends.{backend}.{backend}_{plot_module}".format(
```

Comment:
@ahartikainen How do you feel about pattern where this looks like

`arviz.plots.backends.{backend}.{plot_name}`

Of course the imports would have to change but curious to get your thoughts

<img width="811" alt="image" src="https://user-images.githubusercontent.com/7213793/70382834-6b198c00-1917-11ea-86c5-81e7e0751a1b.png">

Reply:
That sounds ok